brooklyn-tosca-transformer/src/test/resources/templates/simple-with-yaml-error.yaml

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
This file contains a Yaml indentation error and the output in the console is identical to the previous one that has a relationship referencing a missing node.  Isn't there a way to print an error message that is obvious  about its cause?

Reply:
the error it prints is:

```
org.apache.brooklyn.util.exceptions.PropagatedRuntimeException: Unable to instantiate item: Plan looks like it's meant to be TOSCA but it is not valid YAML: ScannerException: mapping values are not allowed here
 in 'reader', line 16, column 17:
      node_templates:
```